fedimint-api-client/src/api/mod.rs

```diff
@@ -257,7 +256,9 @@ pub trait FederationApiExt: IRawFederationApi {
                         || async {
                             self.request_single_peer(method.clone(), params.clone(), *peer)
                                 .await
-                                .inspect_err(|e| e.report_if_important(*peer))
+                                .inspect_err(|e| {
+                                    e.report_if_unusual(*peer, "QueryWithStrategyRetry");
```

Comment:
[nit] The context is more similar to `request_with_strategy` using `RequestWithStrategyRetry`